### PR TITLE
Code Cleanup

### PR DESCRIPTION
**Pull request by shundhammer**
## Description

This cleans up some cruft that has accumulated over the years:

- Removed pointless logging that was good for developing a specific piece of code, but that now just spams the y2log
- Include headers in the new Qt style and consistently: `#include <QFooWidget>` instead of `#include <qfoowidget.h>`
- Follow our C++ coding style
- Don't use cryptic variable names to save typing 2 characters
- Fixed misleading comments or broken English in comments

## Version number? Change log?

None; it will go to OBS with the next PR.

### CODE REVIEW

File: libyui/src/YDialogSpy.cc [hunk 1]
```diff
@@ -626,73 +653,86 @@ void YDialogSpyPrivate::deleteWidget()
...
+ *
  * @param true = up move to the begining (up/left), false = to the end (down/right)
  */
-void YDialogSpyPrivate::moveSelected(Direction direction)
+void YDialogSpyPrivate::moveSelected( Direction direction )
 {
     auto target_widget = selectedWidget();
-    if (!target_widget) return;
+
+    if ( !target_widget )
```
Comment (wfeldt) [inline libyui/src/YDialogSpy.cc:703]: We mandate a space after unary operators?

Comment (shundhammer) [reply libyui/src/YDialogSpy.cc:703]: Right; fixing.

Comment (shundhammer) [reply libyui/src/YDialogSpy.cc:703]: There is probably still cruft left over in any of those files. It will get less over time.

**Review by wfeldt — APPROVED** [review]
lgtm

**Pull request merged**